docs/learn/smart-contract-developers/create-lsp7-token.md

```diff
@@ -62,17 +61,13 @@ We are now ready to build our contracts using the command:
 npm run build
 ``` -->
 
-## Deploy our LSP7 Token contract on LUKSO Testnet
-
-We are now ready to deploy our contract on the [**LUKSO Testnet network**](../../networks/testnet/parameters)!
+## Deploy your LSP7 Token contract on the LUKSO Testnet
 
-In order to deploy the contract, we will have to update the `hardhat.config.ts` and create a deploy script. Let's go!
+You are now ready to deploy your contract on the [**LUKSO Testnet network**](../../networks/testnet/parameters).
```

Comment:
```suggestion
You are now ready to deploy your contract on the [**LUKSO Testnet network**](../../networks/testnet/parameters.md).
```

Don't forget `.md` on internal links. You can check if you missed it by running `npm run build` --> it should complain.

Reply:
i did that,this one for some reason works only without the .md, even though for most of the guides the opposite is true